### ISSUE TITLE
rocket_get_purge_urls() is adding a double / to always purged URLs

### ISSUE THREAD

**Issue by vmanthos** [wp-media/wp-rocket#5295]
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version ✅ `3.11.5`
- Used the search feature to ensure that the bug hasn’t been reported before ✅

**Describe the bug**

When a URL is added in the **Always Purge URL(s)**, we purge that whenever the cache is cleared. 

We collect those URLs here:
https://github.com/wp-media/wp-rocket/blob/5afc9584b9e57e550231f5a8bfe8ae91fbd0b0cf/inc/common/purge.php#L102-L131

The URLs have a double `//` because of how we handle them on this line:
https://github.com/wp-media/wp-rocket/blob/5afc9584b9e57e550231f5a8bfe8ae91fbd0b0cf/inc/common/purge.php#L129

A URL like this:
`/my-blog-page/`

becomes:
`https://example.com//my-blog-page/`

This prevents preloading the page in the revamped Preload and, although @CrochetFeve0251 will handle such cases, we should make sure the URL is properly formatted.

**To Reproduce**

1. Add a URL in the **Always Purge URL(s)** text area.
2. Log the URLs here: https://github.com/wp-media/wp-rocket/blob/5afc9584b9e57e550231f5a8bfe8ae91fbd0b0cf/inc/common/purge.php#L129
```PHP
error_log( print_r( $purge_urls, true ). "\n", 3, dirname( __FILE__ ) . "/CLEAR.log" );
```

**Expected behavior**

The URL(s) should be properly formatted.

**Additional context**

I believe using `untrailingslashit( $home_url ) .  $page` should fix this.

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by remyperona** [comment]
Seems to be fixed in #5226

**Comment by vmanthos** [comment]
> Seems to be fixed in https://github.com/wp-media/wp-rocket/pull/5226

Indeed! I'm closing this.
